stellarium-improve-appdata.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/data/stellarium.appdata.xml b/data/stellarium.appdata.xml
+index d5813421f3..bab36f1a6e 100644
+--- a/data/stellarium.appdata.xml
++++ b/data/stellarium.appdata.xml
+@@ -401,6 +401,9 @@
+     <category>Education</category>
+     <category>Science</category>
+   </categories>
++  <releases>
++    <release date="2018-07-01" version="0.18.1"/>
++  </releases>
+   <screenshots>
+     <screenshot type="default">
+       <image>https://www.stellarium.org/img/screenshots/0.10-constellation-art-on.jpg</image>
+@@ -418,6 +421,35 @@
+       <image>https://stellarium.org/img/screenshots/0.10-orion-nebula.jpg</image>
+     </screenshot>
+   </screenshots>
++  <content_rating type="oars-1.1">
```

Comment:
Not a blocker, but there's a new compact notation you can use now:
>   <content_rating type="oars-1.1" />
That's the same as all none.